### ISSUE TITLE
wireframes of the homepage + the badgepage

### ISSUE THREAD

**Issue by zzzndr** [mozilla/mozbadging#30]
Here are links to several wireframes of the homepage + the badgepage

Wireframe homepage, logged in: 
https://redpen.io/wsb8297eec70aa03df

Wireframe homepage, search field + help popup:
https://redpen.io/fva8dc22b6a5e520ac

Wireframe homepage, user dashboard: 
https://redpen.io/mte2f31f671414588f

Wireframe badgepage
https://redpen.io/kv6df4dc54adc74d63

Looking forward to everybody’s feedback!

@emgollie @christensenep @andrewhayward @adamlofting @davidascher

**Comment by emgollie** [comment]
Thanks @zzzndr! Look forward to discussing tomorrow, and would like to invite @chloeatplay @iamjessklein @MatthewW @cassiemc to provide feedback on the Red Pen links as well.

Most of my global page feedback can be found on https://redpen.io/wsb8297eec70aa03df.

**Comment by emgollie** [comment]
@zzzndr posted two new wireframes for team comment, @christensenep @andrewhayward @adamlofting @davidascher @chloeatplay:

Issuer logged in page: https://redpen.io/rt9ca298e7befe3269
Earner logged in page: https://redpen.io/qpd50d0b0780f16ed7

I like that splitting these two use cases would allow us to simplify the homepage. Thoughts?

**Comment by davidascher** [comment]
yes, this feels way more approachable.  general +1 on that approach.

Makes me want to ask what the team view looks like -- in particular, graphs
of growth rate of badge issuances are likely more important to show status
than just a list of recent issuees (although those are useful too).

**Comment by ghost** [comment]
Yeah, I like the split use cases, and also some of the other changes you've made there.  In particular I like the change to the "recently awarded" list to make it list only one individual's badge awards at a time.  Makes it feel more... playful?  As opposed to a big dump of information demanding to be processed.  Or something.
